### PR TITLE
Changed login page and logo 

### PR DESCRIPTION
**Pull request by BreanneHuang**
- [x] Designed a new logo

- [x] Updated login page to match the universal colour and font scheme

### CODE REVIEW

File: app/src/containers/LoginScreen.tsx [hunk 1]
```diff
@@ -80,28 +80,32 @@ class LoginScreen extends Container<ILoginScreenProps, ILoginScreenState> {
...
 			<div
 				className={"fill-container center-mid"}
 			>
 				<div style={{
-					marginBottom: 180,
+					marginBottom: 15,
+					marginRight: 100
 				}}>
-					<Image path={"/logo.png"} width={600} height={140}/>
+					<img src={"/musico.svg"} alt="Logo" width={800} height={500} />
```
Comment (perryliao) [inline app/src/containers/LoginScreen.tsx:94]: why are you using `` instead of `<Image/>` ?

Comment (BreanneHuang) [reply app/src/containers/LoginScreen.tsx:94]: Idk how to change the “object-fit” from “cover” to “contain” using <Image />

File: app/src/utility/constants.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
-export const API_URL = process.env.REACT_APP_API_URL;
+export const API_URL = "http://localhost:9000/";
```
Comment (aditya10) [inline app/src/utility/constants.ts:1]: NOOOOOOO

File: app/src/icons/genres/index.ts [hunk 3]
```diff
@@ -8,10 +8,11 @@ import rockIcon from "./rock.png";
 import reggaeIcon from "./reggae.png";
 import countryIcon from "./country.png";
 import hiphopIcon from "./hiphop.png";
+import indieIcon from "./indie.png";
```
Comment (aditya10) [inline app/src/icons/genres/index.ts:11]: also change latinAmericanIcon to latinIcon

File: app/src/utility/constants.ts (hunk 2 above)
Comment (BreanneHuang) [reply app/src/utility/constants.ts:1]: WHOOPS

**Pull request merged**